src/Adapter/Customer/Group/Repository/GroupRepository.php

```diff
@@ -82,6 +85,38 @@ public function assertGroupExists(GroupId $groupId): void
      */
     public function create(CustomerGroup $customerGroup): GroupId
     {
-        return new GroupId($this->addObjectModelToShops($customerGroup, $customerGroup->id_shop_list, CannotAddGroupException::class));
+        $groupId = $this->addObjectModelToShops(
+            $customerGroup,
+            array_map(fn (int $shopId) => new ShopId($shopId), $customerGroup->id_shop_list),
+            CannotAddGroupException::class
+        );
+
+        return new GroupId($groupId);
+    }
+
+    /**
+     * @param int $customerGroupId
+     *
+     * @return int[]
+     */
```

Comment:
unecessary php doc I guess

Reply:
Actually it's to indicate that the array is an array of int no ShopId